src/proto/streams/prioritize.rs

```diff
@@ -347,13 +347,20 @@ impl Prioritize {
     /// Reclaim just reserved capacity, not buffered capacity, and re-assign
     /// it to the connection
     pub fn reclaim_reserved_capacity(&mut self, stream: &mut store::Ptr, counts: &mut Counts) {
-        // only reclaim requested capacity that isn't already buffered
-        if stream.requested_send_capacity as usize > stream.buffered_send_data {
-            let reserved = stream.requested_send_capacity - stream.buffered_send_data as WindowSize;
+        let span = tracing::trace_span!("reclaim_reserved_capacity");
```

Comment:
Is this span still useful after the fix? Should it be kept?

Reply:
Nope, will remove.